web_src/js/index.js

```diff
@@ -198,4 +199,6 @@ $(document).ready(() => {
   initUserAuthWebAuthnRegister();
   initUserSettings();
   initViewedCheckboxListenerFor();
+
+  initTagNameEditor();
```

Comment:
IMO we do not need to make more global `init` calls.

The `tag-name-editor` only appers on the `repository new release` page.

So the `initRepoReleaseEditor` could be refatored to something like:

```js
export function initRepoReleaseNew() {
    const $repoReleaseNew = $('.repository.new.release');
    if (!$repoReleaseNew.length) return;

    // init tag name editor
    // init content editor
}
```